splinterd/src/daemon.rs

```diff
@@ -453,6 +455,15 @@ impl SplinterDaemon {
             }
         }
 
+        #[cfg(feature = "scabbard-receipt-store")]
+        run_receipt_store_migrations(&self.db_url.to_string()).map_err(|err| {
```

Comment:
This should be run with the `splinter database migrate` command, not here in the daemon

Reply:
Moved this functionality to be run with `splinter database migrate` if "scabbard-receipt-store" is enabled